README.md

```diff
@@ -50,152 +48,76 @@ are automatically reflected on preliminary.istio.io.
 - The content of istio.io is taken from the latest release-XXX branch. The specific branch that
 is used is determined by the istio.io [Netlify](https://netlify.com) project's configuration.
 
-- The content of archive.istio.io is taken from the older release-XXX branches. The set of branches that
-are included on archive.istio.io is determined by the `TOBUILD` variable in this
-[script](https://github.com/istio/istio.io/blob/master/scripts/build_archive_site.sh).
-
 ### Publishing content immediately
 
 Checking in updates to the master branch will automatically update preliminary.istio.io, and will only be reflected on
 istio.io the next time a release is created, which can be several weeks in the future. If you'd like some changes to be
 immediately reflected on istio.io, you need to check your changes both to the master branch and to the
-current release branch (named release-XXX such as release-1.4).
+current release branch (named release-XXX such as release-1.7).
```

Comment:
Are releases always just named with 2 digits? If so maybe this should read `named release-<MAJOR>.<MINOR>`?

Reply:
```suggestion
current release branch (named `release-<MAJOR>.<MINOR>` such as `release-1.7`).
```
I believe so. Yeah, this should look clearer.